src/utils/ctl/cart_ctl.c

```diff
@@ -577,6 +577,18 @@ ctl_register_ctl(crt_endpoint_t *ep)
 	return crt_register_proto_ctl(ep);
 }
 
+
+#define error_exit(x...)	\
+do {				\
+	fprintf(stderr, x);	\
+	exit(-1);		\
+} while (0);
+
+#define error_warn(x...) \
```

Comment:
(style)  Single statement macros should not use a do {} while (0) loop
(style)  do {} while (0) macros should not be semicolon terminated